lib/hpax.ex

```diff
@@ -31,22 +31,46 @@ defmodule HPAX do
 
   @valid_header_actions [:store, :store_name, :no_store, :never_store]
 
+  @doc """
+  Creates a new HPACK table.
+
+  Same as `new/2` with default options.
+  """
+  @spec new(non_neg_integer()) :: Table.t()
+  def new(max_table_size), do: new(max_table_size, [])
```

Comment:
What about adding a default argument to `new/2` instead:

```elixir
def new(max_table_size, options \\ [])
```

Reply:
I did this initially, but then we cannot use `@doc since`. 🙃